otherlibs/systhreads/st_win32.h

```diff
@@ -49,21 +49,26 @@ typedef LONG st_tid;
 #define Tid_Atomic_Compare_Exchange InterlockedCompareExchange
 #endif
 
-/* Return the identifier for the current thread.  Defined in st_stubs.c.
-   The 0 value is reserved. */
-static intnat st_current_thread_id(void);
-
 /* Thread-local storage associating a Win32 event to every thread. */
 static DWORD st_thread_sem_key;
 
+/* Thread-local storage for the OCaml thread ID from the descriptor. */
```

Comment:
"from the descriptor" is not needed.  "the OCaml thread ID" is fine.

Reply:
Done!